ep/build-velox/src/build_velox.sh

```diff
@@ -63,9 +63,9 @@ function compile {
   fi
   # create libvelox_hive_connector.a for VeloxInitializer.cc
   sed -i 's/OBJECT//' velox/connectors/hive/CMakeLists.txt
-  COMPILE_OPTION="-DVELOX_ENABLE_PARQUET=ON"
+  COMPILE_OPTION="-DVELOX_ENABLE_PARQUET=ON -DVELOX_ENABLE_DUCKDB=ON"
```

Comment:
does this patch really requires duckdb?

Reply:
Yes. There's undefined symbol in libvelox.so
```
ldd -r libvelox.so
...
undefined symbol: _ZN8facebook5velox7parquet13duckdb_reader13ParquetReaderC1ESt10shared_ptrINS0_4dwio6common11InputStreamEERKNS6_13ReaderOptionsE       (cpp/build/releases/libvelox.so)
```
When trying to link udf library libmyudf.so it fails.